src/components/editor/property/PropertyLabelInfoTooltip.jsx

```diff
@@ -14,15 +14,20 @@ const PropertyLabelInfoTooltip = (props) => {
   }, [key])
 
   return (
-    <span data-toggle="popover"
-          data-placement="right"
-          data-container="body"
-          title={props.propertyTemplate.label}
-          data-content={props.propertyTemplate.remark}
-          key={key}
-          id={key}>
+    <a href="#"
+       className="tooltip-heading"
+       tabIndex="0"
+       data-toggle="popover"
+       data-trigger="focus"
```

Comment:
This is the line that does the magic, but it only works inside an `<a>` element, not a `<span>` or `<button>` element.